### DISCUSSION TITLE
Order of plugins autoloading

### DISCUSSION THREAD

**Discussion by k0mar12** [nuxt/nuxt#18081] [Questions] [answered]
Which is order of plugins autoloading?

For example, i have list of plugins

- a.client.js
- a.server.js
- b.server.js
- c.client.js

Is according to alphabetical sort? 
Can i control order?

**Comment by viii47** [comment, marked as answer]
If you need to have a specific order then it works as in Nuxt 2.x -> adding them in the designated order in your nuxt.config

**Reply by k0mar12** [reply]
Thank you, but what about autoloading?)

**Reply by viii47** [reply]
@Komar12124 as far as I know `distDir` is responsible for how they're read, looks like it's alphabetically.
